### ISSUE TITLE
Scope error

### ISSUE THREAD

**Issue by dtex** [nodebots/nodebots-interchange#36]
A user raised [this issue](https://github.com/rwaldron/johnny-five/issues/1517) over in the Johnny-Five repo.

The problem appears to be caused by this block in interchange.js:

````js
try {
    const manifest = JSON.parse(fs.readFileSync(manifest_file));
} catch (e) {
    console.error('Manifest file incorrect');
    throw e;
}


let manifest_objects = (options.useFirmata ? manifest.firmata : manifest.backpack);
````

const is scope limited in try blocks so an error is thrown when trying to read ```manifest.firmata``` or ```manifest.backpack```.

**Comment by ajfisher** [comment]
This is being resolved in the new test build out branch.

**Comment by ajfisher** [comment]
This has been fixed in the new 2.0.0 build. Will close and if there's any follow up then please reopen.
